dolphin/db/sqlalchemy/models.py

```diff
@@ -47,10 +47,11 @@ def to_dict(self):
 class AccessInfo(BASE, DolphinBase):
     """Represent access info required for storage accessing."""
     __tablename__ = "access_info"
-    storage_id = Column(String(128), primary_key=True)
-    hostname = Column(String(128))
-    username = Column(String(128))
-    password = Column(String(128))
+    storage_id = Column(String(36), primary_key=True)
+    host = Column(String(255))
+    port = Column(String(255))
+    username = Column(String(255))
+    password = Column(String(255))
```

Comment:
Acess_info would need 'vendor' and 'model' field, These information is required for the drivermanger to select driver..

Reply:
it's not required here, they will be saved in StorageDevice

Reply:
driver selection logic is based on 'vendor' and 'model' . While registering storage you have model and vendor in request body, so register_storage(context, register_info) interface will have this information . But for other operations like list_volumes(context, storage_id ),  storage_id is enough to select the driver. I don't see any mapping in driver manager withr storage_id -> driver